openedx/core/djangoapps/embargo/models.py

```diff
@@ -127,15 +127,17 @@ class RestrictedCourse(models.Model):
     COURSE_LIST_CACHE_KEY = 'embargo.restricted_courses'
     MESSAGE_URL_CACHE_KEY = 'embargo.message_url_path.{access_point}.{course_key}'
 
-    ENROLL_MSG_KEY_CHOICES = tuple([
-        (msg_key, msg.description)
-        for msg_key, msg in six.iteritems(ENROLL_MESSAGES)
-    ])
+    ENROLL_MSG_KEY_CHOICES = sorted(
+        tuple([
```

Comment:
we were converting list to tuple using tuple(). now sorted will convert it back to list.